db/migrate/20241001141442_add_unique_index_to_admins.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToAdmins < ActiveRecord::Migration[7.1]
+  def change
+    add_index :admins, :email, unique: true
```

Comment:
are we sure in all envs when we merge this down this migration won't fail? staging/separation/sandbox/production? might be worth checking